src/compas_viewer/scene/nurbscurveobject.py

```diff
@@ -11,15 +11,10 @@
 
 
 class NurbsCurveObject(ViewerGeometryObject, GeometryObject):
-    """Viewer scene object for displaying COMPAS NurbsCurve geometry.
+    """Viewer scene object for displaying COMPAS NurbsCurve geometry."""
 
-    See Also
-    --------
-    :class:`compas.geometry.NurbsCurve`
-    """
-
-    def __init__(self, curve: NurbsCurve, **kwargs):
-        super().__init__(geometry=curve, **kwargs)
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
if this is only there to be able to set a type for `self.geometry`, you could also leave this away and set the annotation on a class attribute

```python
class NurbsCurveObject(...):
    geometry: NurbsCurve